views/templates/macros/icons.html.twig

```diff
@@ -0,0 +1,9 @@
+{% macro ok(psBaseUri) %}
+    <img src="{{ psBaseUri }}img/admin/enabled.gif" alt="ok">
+{%  endmacro %}
+{% macro nok(psBaseUri) %}
+    <img src="{{ psBaseUri }}img/admin/disabled.gif" alt="nok">
```

Comment:
What if the variable was set in the parent template? Do we still have to send it as parameter?

Reply:
https://twig.symfony.com/doc/3.x/tags/macro.html

> But as with PHP functions, macros don't have access to the current template variables.

We must pass by a parameter

Reply:
Too bad it is not applicable. Marking as solved.